edit-post/components/plugin-post-publish-panel/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Defines extensibility slot for post-publish sidebar.
+ *
+ * Defines slot to be used by plugins to insert content in the post-publish sidebar
+ * (which appears after a user fully publishes a post).
+ *
+ * @see /edit-post/README.md 
+ *
+ * @file   This files defines the PluginPostPublishPanel extension
+ * @author ChrisShultz
+ * @since  2.6.0
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { Slot, Fill } from '@wordpress/components';
+
+/**
```

Comment:
No need it to put it here if there are no deps.

Reply:
Thanks. I removed the unnecessary `@see /edit-posts/README.md`